example/kotlinlib/linting/1-detekt/detekt-config.yml

```diff
@@ -0,0 +1,773 @@
+build:
```

Comment:
Did you get this file from some example or reference somewhere? If so we should leave a comment at the top so people can see where it came from